lib/rubocop/cop/util.rb

```diff
@@ -33,7 +33,10 @@ def parentheses?(node)
       end
 
       def add_parentheses(node, corrector)
-        if node.arguments.empty?
+        if !node.respond_to?(:arguments)
+          corrector.insert_before(node, '(')
+          corrector.insert_after(node, ')')
```

Comment:
Can you use `corrector.wrap(node, '(', ')')` instead of `insert_before` and `insert_after`?